### DISCUSSION TITLE
UnityWebRequest가 Error가 납니다.

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#704] [5. Q&A]
Zepeto.World 버전: 1.3.0

에러 내용:
Exception: file:///D:/_Unity/App/isityou_zepeto/Assets/Script/Manager/ManagerData.js:14: Error: Exception of type 'ZEPETO.Script.Extensions.ForbiddenTypeExtension+ForbiddenException' was thrown.

코드:
![image](https://user-images.githubusercontent.com/5895396/166649660-686f8f6b-6bd7-4c31-9db4-4069332e81ea.png)

혹시 되시는 분 샘플 같은 것 올려주실 수 있나요

**Comment by Joyounwoo** [comment]
현재 제페토에서 외부 링크 사용이 막혀있는 것으로 알고 있습니다.
[제페토 월드 가이드](https://studio.zepeto.me/kr/guides/world-guidelines) 2.5.5 항목을 참고해보시면 좋을 같습니다.

> 2.5.5. 특정 액션 또는 이벤트로 외부 링크에 연결을 시도할 경우 승인 불가
